bin/build-gallery.js

```diff
@@ -328,7 +334,11 @@ function generateGallery (files) {
       </body>
     </html>`
 
-  fs.writeFile('www/gallery.html', html)
+  fs.writeFile('www/gallery.html', html, function(err){
+    if (err) {
+      throw err
+    }
+  })
```

Comment:
Node threw an error, actually, when I first ran the gallery builder, because there was no passed in handler.